### ISSUE TITLE
More Extendable

### ISSUE THREAD

**Issue by calvinmetcalf** [proj4js/proj4js#2]
from @ahocevar's list of todo's on the mainling list

> Make it extensible so developers can add custom transforms, and add
> custom transforms e.g. for Swiss projections, where proj4js only uses
> a 3 param transform because it does not do grid transforms.

**Comment by Feofilakt** [comment]
Hello! Is it still not implemented? Does proj4js currently support custom transforms?

**Comment by ahocevar** [comment]
I do not think this is needed any more. If you have a custom transform, you can use it without proj4. What is your use case @Feofilakt ?

**Comment by Feofilakt** [comment]
I use GeoServer and there is a [custom transform](https://docs.geoserver.org/stable/en/user/configuration/crshandling/coordtransforms.html#define-a-custom-coordinate-operation) for two of spatial references. And I wanted to know if its possible to copy that transform definition to proj4js so it would reproject coordinates the same way as geoserver.

**Comment by ahocevar** [comment]
Unfortunately that's not possible. Datum transforms with grids, grid shifts and lat/lon difference files are not supported.
